programs/locked-voter/src/instructions/freeze_escrow.rs

```diff
@@ -0,0 +1,50 @@
+use crate::*;
+/// Accounts for [voter::freeze_escrow].
+#[derive(Accounts)]
+pub struct FreezeEscrow<'info> {
+    /// [Escrow].
+    #[account(mut,
+        has_one = owner)]
+    pub escrow: Box<Account<'info, Escrow>>,
+
+    /// Authority of the [Escrow].
+    pub owner: Signer<'info>,
+}
+
+impl<'info> FreezeEscrow<'info> {
+    pub fn freeze_escrow(&mut self, freeze_until: i64) -> Result<()> {
```

Comment:
- freeze_until can't be defined by escrow owner
- We should rename this function to `open_dispute`
- When user call this function, there is only 1 purpose is their escrow get hacked, they need multisig wallet to resolve this, so all the parameters must be standardized from locker parameters. 
- They need to provide necessary parameters like: the new owner or token account will receive tokens after multisig resolve